document-generator-prosecution/src/test/java/uk/gov/companieshouse/document/generator/prosecution/ProsecutionDocumentInfoServiceTest.java

```diff
@@ -1,101 +1,91 @@
 package uk.gov.companieshouse.document.generator.prosecution;
 
 import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
 import static org.mockito.Mockito.when;
-import org.junit.jupiter.api.Assertions;
 import org.junit.jupiter.api.DisplayName;
 import org.junit.jupiter.api.Test;
 import org.junit.jupiter.api.TestInstance;
-import org.junit.jupiter.api.TestInstance.Lifecycle;
 import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
 import org.mockito.Mock;
 import org.mockito.junit.jupiter.MockitoExtension;
+import uk.gov.companieshouse.api.model.prosecution.prosecutioncase.ProsecutionCaseStatusApi;
 import uk.gov.companieshouse.document.generator.interfaces.exception.DocumentInfoException;
 import uk.gov.companieshouse.document.generator.interfaces.model.DocumentInfoRequest;
 import uk.gov.companieshouse.document.generator.interfaces.model.DocumentInfoResponse;
-import uk.gov.companieshouse.document.generator.prosecution.UltimatumDocumentInfoBuilderProvider.UltimatumDocumentInfoBuilder;
+import uk.gov.companieshouse.document.generator.prosecution.exception.HandlerException;
+import uk.gov.companieshouse.document.generator.prosecution.handler.ProsecutionHandler;
+import uk.gov.companieshouse.document.generator.prosecution.mapping.model.ProsecutionDocument;
 import uk.gov.companieshouse.document.generator.prosecution.mapping.model.prosecutioncase.ProsecutionCase;
-import uk.gov.companieshouse.document.generator.prosecution.tmpclient.ProsecutionClient;
-import uk.gov.companieshouse.document.generator.prosecution.tmpclient.SdkException;
 
 @ExtendWith(MockitoExtension.class)
-@TestInstance(Lifecycle.PER_CLASS)
+@TestInstance(TestInstance.Lifecycle.PER_CLASS)
 class ProsecutionDocumentInfoServiceTest {
-    private static final String REQUEST_ID = "id1";
-    private static final String PROSECUTION_CASE_URI = "/company/006/prosecution-cases/dd908";
-    private static final String ULTIMATUM_RESOURCE_URI =
-                    "/prosecution/ultimatum" + PROSECUTION_CASE_URI;
-    private static final String SJP_RESOURCE_URI = "/prosecution/sjp" + PROSECUTION_CASE_URI;
 
-    @Mock
-    private UltimatumDocumentInfoBuilderProvider docInfoBuilderProvider;
-
-    @Mock
-    private UltimatumDocumentInfoBuilder docInfoBuilder;
+    @InjectMocks
+    ProsecutionDocumentInfoService prosecutionDocumentInfoService;
 
     @Mock
-    ProsecutionClient prosecutionClient;
+    ProsecutionHandler mockHandler;
 
-    @Mock
-    ProsecutionCase prosecutionCase;
+    private String resourceUri = "uri";
+    private String requestId = "1";
```

Comment:
Are these constants? Should the be made static and final?

Reply:
Done